aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -246,5 +247,37 @@ Tensor& logical_xor_(Tensor& self, const Tensor& other) {
   return native::logical_xor_out(self, self, other);
 }
 
+Tensor& lt_out(Tensor& result, const Tensor& self, const Tensor& other) {
+  TORCH_CHECK(result.scalar_type() == kBool || result.scalar_type() == kByte,
+      "The output tensor of lt must be a bool or byte tensor.");
```

Comment:
Do we still retain the warning somehow for byte output?

```
In [21]: torch.lt(ints, ints, out=byte)
../aten/src/ATen/native/LegacyDefinitions.cpp:100: UserWarning: torch.lt received 'out' parameter with dtype torch.uint8, this behavior is now deprecated,please use 'out' parameter with dtype torch.bool instead.
  "please use 'out' parameter with dtype torch.bool instead.");
Out[21]: tensor([0, 0, 0, 0], dtype=torch.uint8)
```

Reply:
I removed this warning based on @izdeby suggestion